### PR TITLE
Align the sonic-mgmt code based on the typo  "neighbhor"  fix in the image

### PR DESCRIPTION
**Pull request by nhe-NV**
The typo "neighbhor" ip bgp summary is fixed in the https://github.com/sonic-net/sonic-utilities/pull/3375, need to align the test case. 

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

### CODE REVIEW

File: tests/stress/utils.py [hunk 1]
```diff
@@ -22,7 +22,11 @@ def check_queue_status(duthost, queue):
     bgp_neighbors = duthost.show_and_parse(SHOW_BGP_SUMMARY_CMD)
```
Comment (lipxu) [inline tests/stress/utils.py:22]: Could you please help to check whether there are other scripts that use the output of this cmd? thanks

Comment (nhe-NV) [reply tests/stress/utils.py:22]: Updated one more place, this should be all the place related to the typo

**Review by lipxu — APPROVED** [review]
LGTM

**Comment by bingwang-ms** [comment]
The PR for fixing the typo has been reverted due to the potential risk. But I think we are still good to merge this PR as both spells are handled.

**Review by bingwang-ms — APPROVED** [review]

**Pull request merged**